src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,90 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int INITIAL_DEFAULT_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLICATION_FACTOR = 2;
+    private int size;
+    private Node<K, V>[] table;
+
+    MyHashMap() {
+        table = new Node[INITIAL_DEFAULT_CAPACITY];
```

Comment:
Use of raw types is discouraged. You should use the diamond operator to create an instance of the Node array: `table = new Node<>[INITIAL_DEFAULT_CAPACITY];`